server/api/controller/cron.js

```diff
@@ -210,6 +208,29 @@ const sendTreeAlerts = () => {
 		});
 };
 
+const updatePlanTags = async () => {
+	let start = Number(Date.now());
+	Log.info('Re-creating plan tags');
+	let tagCounter = 0;
+	// Re-compute the tags of a plan if the last update time of the plan is after the last update time of the tags of this plan.
+	// Before re-computing the tags of a plan, remove all previous tags for this plan.
+	
+	// TODO: Loop on the plans that need to be updated 
+	const plans = await Plan.getPlansToTag();
```

Comment:
Do we set limits on how many plans we fetch to extract tags from?

Reply:
After discussing with @TomerMe2 I am going to implement the following logic in getPlansToTag: 
1. add an indicator of when we last updated the tags to the plan table
2. update tags last updated indicator when we update tags
3. compare it to when the plan was last updated in this function. 

Do you want to also set a max number of records?

Reply:
Hi Michal
As for now, Tomer explained to me that the tag generation is an easy task, and doesn't require loads of processing. Anyways, I think it would be better if we do manage the number of plans we generate tags for, as our server has no resources

Reply:
CC @TomerMe2 when you come back. I am talking to @GalGend and she says that we don't handle plan updates yet, so maybe this is not necessary yet?